.github/workflows/pre-release.yaml

```diff
@@ -0,0 +1,108 @@
+name: "Pre-Release"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version to update.
+        required: true
+      tracker-url:
+        type: string
+        description: The URL to tracker issue.
+        required: true
+env:
+  VERSION: ${{ inputs.version }}
+  TRACKER_URL: ${{ inputs.tracker-url }}
+jobs:
+  dry-run-pr:
+    runs-on: ubuntu-latest
+    name: Create dry-run pr and update tags
+    steps:
+      - uses: actions/checkout@v4
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+      - name: Validate semver
+        run: ./.github/scripts/validate-semver.sh v${{ env.VERSION }}
+      - uses: ./.github/actions/update-manifest-branches
+      - name: Create dry-run pr
+        uses: peter-evans/create-pull-request@v6
+        id: cpr-dry-run
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          commit-message: "Test ${{ env.VERSION }} Release"
+          branch: odh-release/e2e-dry-run
+          delete-branch: true
+          title: "[DO NOT MERGE] Test ${{ env.VERSION }} Release"
+      - name: Wait for checks to pass
+        env:
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+        run: ./.github/scripts/wait-for-checks.sh ${{ steps.cpr-dry-run.outputs.pull-request-number }}
+      - name: Close PR
+        uses: peter-evans/close-pull@v3
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          pull-request-number: ${{ steps.cpr-dry-run.outputs.pull-request-number }}
+          comment: Auto-closing pull request after success checks
+          delete-branch: true
+      - name: Push version tag to quay.io
+        run: |
+          skopeo login -u ${{ secrets.QUAY_ID }} -p ${{ secrets.QUAY_TOKEN }} quay.io/{{ secrets.QUAY_ORG }}
+          skopeo copy docker://quay.io/{{ secrets.QUAY_ORG }}/opendatahub-operator:pr-${{ steps.cpr-dry-run.outputs.pull-request-number }} docker://quay.io/{{ secrets.QUAY_ORG }}/opendatahub-operator:${{ env.VERSION }}
+          echo "Successfully updated tag to quay.io with version: ${{ env.VERSION }}"
```

Comment:
https://github.com/marketplace/actions/skopeo-copy-action ?

Reply:
I saw that action, but looked like a very unsupported action 😅  so I did not use it. Do you think it is okay to use?

Reply:
ok, keep your part, i am fine with this.

Reply:
looking at the code, https://github.com/coscene-io/skopeo-copy-action/blob/main/action.yml
they do pretty much what we do manually